lib/Runtime/Base/JnDirectFields.h

```diff
@@ -651,33 +651,14 @@ ENTRY(toLength)
 ENTRY(toInteger)
 ENTRY(arraySpeciesCreate)
 ENTRY(arrayCreateDataPropertyOrThrow)
-ENTRY(Array_values)
-ENTRY(Array_keys)
-ENTRY(Array_entries)
-ENTRY(Array_indexOf)
-ENTRY(Array_sort)
-ENTRY(Array_filter)
-ENTRY(Array_flat)
-ENTRY(Array_flatMap)
-ENTRY(Array_forEach)
-ENTRY(Array_some)
-ENTRY(Array_every)
-ENTRY(Array_includes)
-ENTRY(Array_reduce)
-ENTRY(Object_fromEntries)
-ENTRY(Math_min)
-ENTRY(Math_max)
-ENTRY(FunctionKind)
 
 // EngineInterfaceObject built-ins
-ENTRY(builtInJavascriptArrayEntryFilter)
-ENTRY(builtInJavascriptArrayEntryForEach)
-ENTRY(builtInJavascriptArrayEntryIndexOf)
-ENTRY(builtInJavascriptArrayEntrySome)
-ENTRY(builtInJavascriptArrayEntryEvery)
-ENTRY(builtInJavascriptArrayEntryIncludes)
-ENTRY(builtInMathMin)
-ENTRY(builtInMathMax)
+ENTRY(builtInArray_prototype_filter)
+ENTRY(builtInArray_prototype_forEach)
+ENTRY(builtInArray_prototype_indexOf)
+ENTRY(builtInArray_prototype_reduce)
+ENTRY(builtInMath_object_min)
+ENTRY(builtInMath_object_max)
```

Comment:
These Entries were renamed to work slightly better with my re-worked process below. Other entries were no longer needed hence deleted.